mapit_gb/management/commands/mapit_UK_add_slugs_to_local_authorities.py

```diff
@@ -0,0 +1,38 @@
+# This script is for a one off import of govuk slugs for local authorities
+
+import json
+from django.core.management.base import NoArgsCommand
+from mapit.models import Area, CodeType
+from django.utils.six.moves import urllib
+
+
+class Command(NoArgsCommand):
+    help = 'Assigns slugs from frontend to local authorities'
+
+    def handle_noargs(self, **options):
+        try:
+            code_type = CodeType.objects.get(code='govuk_slug')
+        except CodeType.DoesNotExist:
+            code_type = CodeType(
+                code='govuk_slug',
+                description='Slug for use by GOV.UK'
+            )
+            code_type.save()
```

Comment:
We wanted to avoid clashing with any future changes that mysociety might make - both in the fixture file itself and in the primary keys.  Also it means you can run this script without having to remember to run loaddata uk first to make sure the codetype is present.